scripts/hostcfgd

```diff
@@ -1491,6 +1491,27 @@ class DeviceMetaCfg(object):
         syslog.syslog(syslog.LOG_INFO, 'DeviceMetaCfg: Restart rsyslog after '
                       'changing timezone')
 
+    def rsyslog_config(self, data):
+        """
+        Apply syslog_with_osversion feature flag.
+        Run the following command in Linux: sudo systemctl restart rsyslog-config
+        Args:
+            data: Read table's key's data.
+        """
+        syslog_with_osversion = data.get('syslog_with_osversion')
+        syslog.syslog(syslog.LOG_DEBUG,
+                      f'DeviceMetaCfg: syslog with os version: {syslog_with_osversion}')
+
+        if syslog_with_osversion is None:
```

Comment:
>syslog_with_osversion [](http://example.com/codeflow?start=11&length=21)

Should we check the new config with current running config, if only changed, restart service ?

Reply:
Fixed